AdyenCard/Components/Stored Card/StoredCardComponent.swift

```diff
@@ -27,9 +29,11 @@ internal final class StoredCardComponent: PaymentComponent, PaymentAwareComponen
     private let storedCardPaymentMethod: StoredCardPaymentMethod
     
     internal init(storedCardPaymentMethod: StoredCardPaymentMethod,
+                  payment: Payment? = nil,
```

Comment:
let's not use default nil values, because we don't want to miss passing it by mistake :- )